client/src/components/Signup.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import EmailField from './EmailField';
+import UsernameField from './UsernameField';
+import PasswordField from './PasswordField';
+import FormDialog from './FormDialog';
+import useSignupForm from '../customHooks/useSignupForm';
+
+const Signup = ({ toggleModal, show }) => {
+  const closeSignupModal = () => toggleModal('showSignup');
+
+  const {
+    emailState,
+    usernameState,
+    passwordState,
+    repeatPasswordState,
+    handleSubmit,
+  } = useSignupForm(closeSignupModal);
+
+  return (
+    <FormDialog
+      title="Sign Up"
+      show={show}
+      closeModal={closeSignupModal}
+      handleSubmit={handleSubmit}
+    >
+      <EmailField autoFocus {...emailState} />
+      <UsernameField {...usernameState} />
+      <PasswordField {...passwordState} />
+      <PasswordField {...repeatPasswordState} name="repeat-password" label="Repeat Password" />
+    </FormDialog>
+  );
+};
+
+Signup.propTypes = {
+  show: PropTypes.bool.isRequired,
+  toggleModal: PropTypes.func.isRequired,
+};
+
+Signup.displayName = 'Signup';
```

Comment:
Whats this for?

Reply:
Yeah, I was trying to get `enzyme` (testing library) to detect the Signup component but turned out this wasn't really needed and I had just forgotten to remove it. I'll remove it now. :)